azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/ChatCompositeMessage.kt

```diff
@@ -0,0 +1,67 @@
+package com.azure.android.communication.ui.chat.presentation.ui.chat.components
```

Comment:
missing copyright

Reply:
added copyright notices in files missing it.

Reply:
wait, can't push out of date somehow

Reply:
ok, updated now